CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   Setup mocha and nyc for tests and coverage
 -   Setup basic coding style tools
 -   Write complete contributing guide
+-   Add rule `@ecocode/limit-db-query-results`
```

Comment:
Can you put this line in the `unreleased` > `Added` section above? Instead of here